resolwe/test/testcases.py

```diff
@@ -281,7 +281,7 @@ def setUp(self):
         self._register_schemas()
 
         self.collection = Collection.objects.create(contributor=self.admin, name="Test collection")
-        self.files_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files')
+        self.files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'toolkit', 'tests', 'files'))
```

Comment:
I also do not like that this breaks the independency of apps. There is no good general way to support this though. I would have to override the `self.files_path` in every single test class.